### ISSUE TITLE
prompt-toolkit version detection error

### ISSUE THREAD

**Issue by GuillaumeLeclerc** [scopatz/xonsh#843]
Hello,

Thank you for this tool. I'd really like to test it but I get a warning each time I start it I get a warning that tells me to update prompt-toolkit even if I just installed through PIP 10 minutes ago. 

How can I solve it ? 

Thanks

**Comment by scopatz** [comment]
Hi @GuillaumeLeclerc - thanks for reporting. Can you post the full error, please? Also what is the output of the `xonfig` command?

**Comment by GuillaumeLeclerc** [comment]
Sorry the name of the issue was wrong

```
/usr/local/lib/python3.4/dist-packages/xonsh/shell.py:97: RuntimeWarning: prompt-toolkit version < v0.57 and may not work as expected. Please update.
  warn(msg, RuntimeWarning)
```

I checked my PIP packages and I have the version and I have v1.0. maybe the change in the major version messed up the version detector ?

**Comment by azag0** [comment]
This is fixed in current HEAD which supports 1.0.0. Prompt-toolkit probably changed their version reporting, so it broke the version detection in olders xonshes.

**Comment by GuillaumeLeclerc** [comment]
Ok thanks. I'll wait the next release it's not dealbreaking. 

Thanks for your awesome work !
